packages/integration-tests/src/tests/api/profile/index.test.ts

```diff
@@ -122,6 +123,46 @@ describe('profile', () => {
       await deleteDefaultTenantUser(user.id);
       await deleteDefaultTenantUser(user2.id);
     });
+
+    it('should be able to update profile', async () => {
+      const { user, username, password } = await createDefaultTenantUserWithPassword();
+      const api = await signInAndGetUserApi(username, password);
+      const newProfile = {
+        profile: 'HI',
+        address: {
+          country: 'USA',
+        },
+      };
```

Comment:
Any existing profile data should be overwritten by the new ones

Reply:
No, it is partial